python/py-pivy/Portfile

```diff
@@ -14,7 +14,7 @@ description         Pivy is a Coin binding for Python.
 long_description    ${description}
 platforms           darwin
 
-python.versions     27 35 36
+python.versions     27 35 36 37 38 39 310
```

Comment:
are the old Python subports still used anywhere as dependents or could they be removed?

Reply:
@reneeotten `py27` is used by freecad ;)

Reply:
@reneeotten it is used only by freecad. I may remove old versions without any problems.

Reply:
done